src/material-experimental/mdc-select/select.scss

```diff
@@ -1 +1,115 @@
-// TODO: implement MDC-based select
+@import '@material/menu-surface/mixins.import';
+@import '@material/list/mixins.import';
+@import '../../material/core/style/variables';
+@import '../../material/core/style/vendor-prefixes';
+@import '../../cdk/a11y/a11y';
+
+$mat-select-arrow-size: 5px !default;
+$mat-select-arrow-margin: 4px !default;
+$mat-select-panel-max-height: 256px !default;
+
+$mat-select-placeholder-arrow-space: 2 * ($mat-select-arrow-size + $mat-select-arrow-margin);
+
+@include mdc-menu-surface-core-styles($query: structure);
+
+.mat-mdc-select {
```

Comment:
Normally we write a comment explaining each override/additional style we do on top of MDC, in this case I would just leave one at the top here explaining what part of the select styles we take from MDC and what part we just style ourselves because there is no MDC equivalent